packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -92,18 +91,11 @@ export default class HAPIService {
    * Constructs an instance of the class, initializes configuration settings, and sets up various services.
    *
    * @param {Logger} logger - The logger instance used for logging.
-   * @param {Registry} register - The registry instance for metrics and other services.
    * @param {HbarLimit} hbarLimiter - The Hbar rate limiter instance.
    * @param {CacheService} cacheService - The cache service instance.
    * @param {EventEmitter} eventEmitter - The event emitter instance used for emitting events.
    */
-  constructor(
-    logger: Logger,
-    register: Registry,
-    hbarLimiter: HbarLimit,
-    cacheService: CacheService,
-    eventEmitter: EventEmitter,
-  ) {
+  constructor(logger: Logger, hbarLimiter: HbarLimit, cacheService: CacheService, eventEmitter: EventEmitter) {
```

Comment:
A nit, but did you change the formatting on purpose? @ebadiere

Reply:
It looks like he removed the `register: Registry` argument from the args list, so Prettier reformatted it into a single line. It's just a Prettier thing I've seen that a lot.